index.html

```diff
@@ -4,18 +4,34 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="styles.css" />
+    <script
+      src="https://kit.fontawesome.com/2374882fae.js"
+      crossorigin="anonymous"
+    ></script>
 
     <title class="title">Das Frankreich Quiz für Reisefans</title>
```

Comment:
The `<title>` tag should not contain a class attribute and should only contain text. It serves as the document's title in browser tabs and search results.
```suggestion
    <title>Das Frankreich Quiz für Reisefans</title>
```